### PR TITLE
refactor: components -> fragments

### PR DESCRIPTION
**Pull request by gao-sun**
replace some components with fragments to reduce file size

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/vue.mdx [hunk 1]
```diff
@@ -6,8 +6,8 @@ import Admonition from '@theme/Admonition';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-import AppNote from './components/AppNote';
-import SignInNote from './components/SignInNote';
+import AppNote from './fragments/_app-note.mdx';
+import SignInNote from './fragments/_sign-in-note.mdx';
 
 # 集成 `@logto/vue`
```
Comment (demonzoo) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/vue.mdx:13]: I don't see any change in `android.mdx`. Did you miss it?

File: docs/docs/recipes/integrate-logto/fragments/_app-note.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+:::note
+
+<p>
+  This tutorial assumes you have created an Application of type &quot;{props.type}&quot; in Admin
+  Console. If you are not ready,{' '}
+  <a href="../../tutorials/get-started/create-and-integrate-the-first-app">read this</a> one before
+  continuing.
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/fragments/_app-note.mdx:7]: just wonder why add a link break here?
```suggestion
  This tutorial assumes you have created an Application of type &quot;{props.type}&quot; in Admin Console. 
  If you are not ready,{' '}
  <a href="../../tutorials/get-started/create-and-integrate-the-first-app">read this</a> one before
  continuing.
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/fragments/_app-note.mdx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+:::note
+
+<p>
+  本篇教程默认你已经在管理界面中成功创建了一个「{props.type}」类型的
+  应用。如果你还没有完成这一步操作，请参阅{' '}
```
Comment (IceHe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/fragments/_app-note.mdx:5]:
```suggestion
  本篇教程默认你已经在管理界面中成功创建了一个「{props.type}」类型的应用。
  如果你还没有完成这一步操作，请参阅{' '}
```

File: docs/docs/recipes/integrate-logto/fragments/_sign-in-note.mdx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+:::note
+
+<p>
+  Before calling <code>{props.calling}</code>, make sure you have correctly configured Redirect URI
+  in Admin Console.
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/fragments/_sign-in-note.mdx:5]: ditto
```suggestion
  Before calling <code>{props.calling}</code>, make sure you have correctly configured Redirect URI in Admin Console.
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/vue.mdx (hunk 1 above)
Comment (gao-sun) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/vue.mdx:13]: android didn't use these fragments

File: docs/docs/recipes/integrate-logto/fragments/_app-note.mdx (hunk 2 above)
Comment (gao-sun) [reply docs/docs/recipes/integrate-logto/fragments/_app-note.mdx:7]: prettier did it i think

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/fragments/_app-note.mdx (hunk 3 above)
Comment (gao-sun) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/fragments/_app-note.mdx:5]: prettier did it i think

File: docs/docs/recipes/integrate-logto/fragments/_sign-in-note.mdx (hunk 4 above)
Comment (gao-sun) [reply docs/docs/recipes/integrate-logto/fragments/_sign-in-note.mdx:5]: prettier did it i think

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/fragments/_app-note.mdx (hunk 3 above)
Comment (IceHe) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/fragments/_app-note.mdx:5]: maybe up to the element width?

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/vue.mdx (hunk 1 above)
Comment (demonzoo) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/vue.mdx:13]: ok

**Review by demonzoo — APPROVED** [review]
Good to me

**Pull request merged**